clients/cpp-console/include/DialogManager.h

```diff
@@ -44,12 +44,17 @@ class DialogManager
     DialogManager(shared_ptr<AgentConfiguration> agentConfig, string audioFilePath);
     const DeviceStatus GetDeviceStatus() { return _deviceStatus; };
     const KeywordActivationState GetKeywordActivationState() { return _keywordActivationState; }
+    //Start a listening session that will terminate after the first utterance. Audio player will be stopped if barge in is supported.
```

Comment:
Space after the "//". Same for below

Reply:
Regarding "Audio player will be stopped if barge in is supported"  - That's not the right support . TTS playback should stop regardless if barge in is supporet or not. See the [barge in section in the README](https://github.com/Azure-Samples/Cognitive-Services-Voice-Assistant/tree/master/clients#what-does-barge-in-mean) - "Note that in both cases, the user can always press a mic button to trigger a ListenOnceAsync call, which will immediately stop TTS playback and the client will wait for the bot to process the new utterance."

Reply:
Updated.